stm/tacworkertop.ml

```diff
@@ -10,5 +10,5 @@ module W = AsyncTaskQueue.MakeWorker(Stm.TacTask) ()
 
 let () = Coqtop.toploop_init := WorkerLoop.loop W.init_stdout
 
-let () = Coqtop.toploop_run := (fun _ _ -> W.main_loop ())
+let () = Coqtop.toploop_run := (fun _ ~state -> ignore(state); W.main_loop ())
```

Comment:
nitpick, you can write `fun _ ~state:_ -> ....`

Reply:
Oh thanks, I never got the right syntax for this.